src/containers/AppView.js

```diff
@@ -1,32 +1,30 @@
 import React, { Component, PropTypes } from 'react';
-import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
-import * as Actions from '../actions/actions.js';
+import * as actions from '../actions';
 
 class AppView extends Component {
-
   static propTypes = {
-    dispatch: PropTypes.func.isRequired
+    changeLook: PropTypes.func.isRequired
   }
 
   render () {
-    const {dispatch } = this.props;
-    const actions = bindActionCreators(Actions, dispatch);
-
     return (
       <div>
-      <h1 onClick={function() {
-              console.log(actions.changeLook(2))
-              }}>Hello - {this.props.look}</h1>
+        <h1 onClick={() => this.props.changeLook(2)}>
+          Hello - {this.props.look}
+        </h1>
       </div>
     );
   }
 }
+
 function mapStateToProps(state) {
-  debugger;
     return {
-        look: state
+        look: state.look.multipleOutFitLook
     };
 }
 
-export default connect(mapStateToProps)(AppView)
+export default connect(
+  mapStateToProps,
+  actions
```

Comment:
React Redux will bind any action creators passed as second argument and inject them as props.
That said you can keep the old approach—the new one is just less typing.